### ISSUE TITLE
Constant TYPO3_version already defined

### ISSUE THREAD

**Issue by tlueder** [sascha-egerer/phpstan-typo3#65]
In case one uses conditional PHPStan config like:

phpstan.neon

```NEON
includes:
    - vendor/saschaegerer/phpstan-typo3/extension.neon
    - PHPStan/conditional.config.php
```

PHPStan/conditional.config.php

```PHP
<?php

declare(strict_types=1);

use TYPO3\CMS\Core\Information\Typo3Version;

$config = [];

$typo3Version = new Typo3Version();
if (version_compare($typo3Version->getVersion(), '11.5.0', '<')) {
  $config['parameters']['ignoreErrors'][] = [
    'message' => '#Call to an undefined method TYPO3\\\\CMS\\\\Core\\\\Database\\\\Query\\\\QueryBuilder::executeQuery\(\).#',
    'path' => '../Classes/ViewHelpers/path.php',
    'count' => 1,
  ];
}

return $config;
```

PHP Notice is shown:

PHP Notice:  Constant TYPO3_version already defined in vendor/saschaegerer/phpstan-typo3/phpstan.bootstrap.php on line 20

**Comment by sascha-egerer** [comment]
I'm not sure if this should be done in the configuration like that as we're now loading code in the configuration that should only be loaded in the bootstrap. This could lead to other problems deep inside of PHPStan. It could possibly end up in unexcpected behavior like broken caches or so. Not sure if something will break but nobody knows. It just feels wrong doing it like that.
Do you have more details where any why you wan't to do it like that? DO you have a TYPO3 Extension that is compatible with multiple TYPO3 versions? Could you not just add a separate config file that ins used in CI when TYPO3 version is < 11.5?

**Comment by tlueder** [comment]
Yes, I have an extension that supports 9.5 , 10.4 and 11.5 and where it just for CI I wouldn't bother. But this is for lokal testing and having just one command to run that then figures out the rest is just convenient.

But I'm open for suggestions if you have a better way to do this.

**Comment by sascha-egerer** [comment]
How do you trigger you local testing for these TYPO3 versions? You're doing a `composer install` somehow to install the TYPO3 version?

**Comment by tlueder** [comment]
Yes, I found myself switching between 9.5, 10.4 and 11.5 quite a lot, while backporting stuff, my team and I had written for 11.5, we needed in older projects too.

And for triggering the testing, I've mapped a VSCode button to the command "php -d memory_limit=512M vendor/bin/phpstan analyse -c phpstan.neon"

As I said, this is just a convenience thing, but I don't want to mess with phpstan or introduce bugs I then have to look for.

You are right, I could create separate config files and hope to remember to use them. :)
Or I change my approach and figure out which Typo3 version is currently active another way. I have to think about that.
